features/step_definitions/pages_steps.rb

```diff
@@ -63,4 +63,18 @@ def static_page_path(page)
 Then(/^the "([^"]*)" page title should read "([^"]*)"$/) do |str, title|
 	visit path_to(str)
 	expect(page.title).to eq "#{title}"
-end
\ No newline at end of file
+end
+
+Then(/^I see the banners for all sponsors$/) do
+sponsors = {
+  "Standuply" => "https://standuply.com/?utm_source=links&utm_medium=agileventures&utm_campaign=partnership",
+  "Craft Academy" => "http://craftacademy.se/english",
+  "Mooqita" => "https://mooqita.org/",
+  "Amazon Smile" => "https://smile.amazon.co.uk/ch/1170963-0"
+}
+  sponsors.each do |key, value|
+    within("//a[@href='#{value}']/div") do
+      find(:xpath, "//img[@alt='#{key}']")
```

Comment:
Using `find` here I suppose the test will fail if it doesn't find the image on the page right?
It's not immediately clear while reading this that we expect to find an image using this test ... or perhaps it's just me. I'd probably have something along the line of

```ruby
expect(page).to have_selector("img[@alt='#{key}']")
```
What do you think about that?

Reply:
Thanks @diraulo for the review, I also need to change the key-value variable for something which makes more sense.